scheduler/src/cook/mesos/api.clj

```diff
@@ -932,6 +941,24 @@
       (log/debug e "Unable to retrieve directory path for" task-id "on agent" agent-hostname)
       nil)))
 
+(defn fetch-compute-cluster-map
+  "Converts a compute cluster entity as a map representing the fields. For legacy instances
+  that predate cook-cluster, inline the default cluster, pulled from config."
```

Comment:
I thought we had discussed just not returning this at all for jobs that are missing it. How do you feel about that?

Reply:
See comment above. The consistency of having the dictionary always there is nice.